src/components/Alert/__stories__/Alert.stories.tsx

```diff
@@ -12,133 +16,227 @@ export default {
     },
-    {
-        message,
-        theme: 'info',
-        view: 'filled',
-        actions: (
-            <Alert.Actions>
-                <Alert.Action>{center}</Alert.Action>
-            </Alert.Actions>
-        ),
+];
+
+export const Title: Story = {
+    render: (args) => (
+        <Showcase>
+            {titleCases.map(({caseTitle, propValue}, index) => (
+                <ShowcaseItem key={index} title={caseTitle}>
+                    <Alert {...args} title={propValue} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
     },
+};
+
+const messageCases: Array<{caseTitle: string; propValue: AlertProps['message']}> = [
     {
-        title,
-        message,
-        theme: 'info',
-        view: 'outlined',
+        caseTitle: 'Regular message',
+        propValue: Default.args?.message,
     },
-    {
-        title,
-        message,
-        theme: 'success',
-        view: 'outlined',
-        actions: (
-            <Alert.Actions>
-                <Button view="action">{right}</Button>
-                <Button view="outlined">{center}</Button>
-                <Button view="flat">{left}</Button>
-            </Alert.Actions>
-        ),
+];
+
+export const Message: Story = {
+    render: (args) => (
+        <Showcase>
+            {messageCases.map(({caseTitle, propValue}, index) => (
+                <ShowcaseItem title={caseTitle} key={index}>
+                    <Alert {...args} message={propValue} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
     },
-    {
-        title,
-        message,
-        theme: 'success',
-        view: 'filled',
+};
+
+const themeCases: Array<AlertProps['theme']> = [
+    'normal',
+    'info',
+    'success',
+    'warning',
+    'danger',
+    'utility',
+];
 
-        actions: [{text: right, handler: console.log}],
+export const Theme: Story = {
+    render: (args) => (
+        <Showcase>
+            {themeCases.map((theme, index) => (
+                <ShowcaseItem title={theme} key={index}>
+                    <Alert {...args} theme={theme} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
     },
-    {
-        title,
-        message,
-        theme: 'warning',
-        view: 'filled',
-        layout: 'horizontal',
-        corners: 'square',
-        onClose: () => {},
-        actions: (
-            <Alert.Actions>
-                <Alert.Action view="outlined">{left}</Alert.Action>
-            </Alert.Actions>
-        ),
+};
+
+const iconCases: Array<{caseTitle: React.ReactNode; propValue: AlertProps['icon']}> = [
+    {caseTitle: 'Without icon', propValue: null},
+    {caseTitle: <IconComponent size={16} data={Gear} />, propValue: 'With icon'},
+];
+
+export const Icon: Story = {
```

Comment:
We can name it `CustomIcon` and keep single case, no need to show regular case, it's the same as default

Reply:
refactored https://github.com/gravity-ui/uikit/pull/1597/commits/a4819d2bdf8886eafadb5b6f79df18f1f7398f2f